frontend/pages/map.tsx

```diff
@@ -0,0 +1,5 @@
+import dynamic from "next/dynamic"
+
+// Temporary page to display the leaflet map for development
+
+export default dynamic(() => import("../components/Map"), { ssr: false })
```

Comment:
lets move this to the dashboard page since that is where the map will live :)

Reply:
Including the link to D3 here for the sake of discussion. I'll test leaflet out and give some feedback in the next few days.
https://observablehq.com/@d3/gallery

Reply:
Dropping a link to a choropleth I made using d3 so folks can see how it works: https://github.com/dylanesque/Choropleth

Reply:
Thanks all for your comments! See you tonight!